decidim-budgets/lib/decidim/budgets/seeds.rb

```diff
@@ -13,56 +13,76 @@ def initialize(participatory_space:)
           }
         )
+      end
 
-        rand(1...3).times do
-          Decidim::Budgets::Budget.create!(
-            component:,
-            title: Decidim::Faker::Localized.sentence(word_count: 2),
-            description: Decidim::Faker::Localized.wrapped("<p>", "</p>") do
-              Decidim::Faker::Localized.paragraph(sentence_count: 3)
-            end,
-            total_budget: ::Faker::Number.number(digits: 8)
-          )
-        end
-
-        Decidim::Budgets::Budget.where(component:).each do |budget|
-          rand(2...4).times do
-            project = Decidim::Budgets::Project.create!(
-              budget:,
-              scope: participatory_space.organization.scopes.sample,
-              category: participatory_space.categories.sample,
-              title: Decidim::Faker::Localized.sentence(word_count: 2),
-              description: Decidim::Faker::Localized.wrapped("<p>", "</p>") do
-                Decidim::Faker::Localized.paragraph(sentence_count: 3)
-              end,
-              budget_amount: ::Faker::Number.between(from: Integer(budget.total_budget * 0.7), to: budget.total_budget)
-            )
+      def create_budget!(component:)
+        Decidim::Budgets::Budget.create!(
+          component:,
+          title: Decidim::Faker::Localized.sentence(word_count: 2),
+          description: Decidim::Faker::Localized.wrapped("<p>", "</p>") do
+            Decidim::Faker::Localized.paragraph(sentence_count: 3)
+          end,
+          total_budget: ::Faker::Number.number(digits: 8)
+        )
+      end
 
-            attachment_collection = create_attachment_collection(collection_for: project)
-            create_attachment(attached_to: project, filename: "Exampledocument.pdf", attachment_collection:)
-            create_attachment(attached_to: project, filename: "city.jpeg")
-            create_attachment(attached_to: project, filename: "Exampledocument.pdf")
+      def create_project!(budget:)
+        params = {
+          budget:,
+          scope: participatory_space.organization.scopes.sample,
+          category: participatory_space.categories.sample,
+          title: Decidim::Faker::Localized.sentence(word_count: 2),
+          description: Decidim::Faker::Localized.wrapped("<p>", "</p>") do
+            Decidim::Faker::Localized.paragraph(sentence_count: 3)
+          end,
+          budget_amount: ::Faker::Number.between(from: Integer(budget.total_budget * 0.7), to: budget.total_budget)
+        }
 
-            Decidim::Comments::Seed.comments_for(project)
-          end
+        if budget.component.settings.geocoding_enabled?
+          params = params.merge(
+            address: "#{::Faker::Address.street_address} #{::Faker::Address.zip} #{::Faker::Address.city}",
+            latitude: ::Faker::Address.latitude,
+            longitude: ::Faker::Address.longitude
+          )
         end
+
+        Decidim::Budgets::Project.create!(params)
```

Comment:
missing traceability :)

Reply:
Done in [cd4694a](https://github.com/decidim/decidim/pull/12205/commits/cd4694aa8b6c452e8caf280953d668f6f9f62821)